packages/docs/content/components/label.mdx

```diff
@@ -30,48 +28,31 @@ The following CSS variables can be overridden to customize Form components:
+- Don't place links or other interactive text in labels or legends. Those should go in the [hint text](/components/hint).
+- Don't use a field's placeholder attribute as the primary way to label the field.
+
 ### Label
 
-- Each form field should have a `<label>`. Never use a field's placeholder attribute as the primary way to label the field.
 - Each form field should have a clear, brief, and descriptive label.
 - Labels should have a `for` attribute, referencing the corresponding input's unique `id` attribute. Only one label can be associated to each unique form element.
 - Labels should be placed above their fields.
-- Label text should be short and in sentence case.
-- Avoid colons at the end of labels.
-- Don't use a field's placeholder attribute as the primary way to label the field.
 
 ### Legend
 
 - Use a single legend for fieldset (this is required). One example of a common use of fieldset and legend is a question with radio button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.
-- Avoid links in legends when possible. Links inside legends are not read by some screen readers, namely JAWS. If you need a pattern like this, put the link outside of the legend element, but keep all other useful hint text in the legend.
-
-#### Content
-
-- Describe what the user needs to do when the field is selected
-- Use a phrase or complete sentence
-- Start with an action verb and pair it with a noun to help it be clearer. Example: Enter your age.
-- Use the fewest words possible (i.e. don't use "please")
-- Use sentence case
-- Be consistent with punctuation at the end of a phrase or sentence
-- Be consistent with phrase and sentence structure
-
-### Hint text
-
-- Place hint text within the field's `<label>` element.
-- Place hint text where appropriate depending on what content it's helping to explain
-- Use hint text for supporting contextual help, which will always be shown.
-- Hint text should sit between a form label and the input.
-- Use lower visual style (i.e., lighter gray than the form label or the input option) so it won't compete for users' attention with the field label.
-- Use hint text to indicate which inputs are optional when there are more required fields than optional.
-- Hint text can also appear after an input box when the text is being used to show the value someone should be entering in the field (e.g., income amount input box displays text after the input box to highlight the amount should be "monthly income".)
-
-#### Content
-
-- Use when appropriate to support a form label (Goes below the form label in a lighter weight font)
-- Describe how to best answer the question/form label
-- Use the fewest words possible (i.e. don't use "please")
-- Use sentence case
-- End with a period
+
+### Content
+
+- Describe what the user needs to do when the field is selected.
+- Use a phrase or complete sentence.
```

Comment:
do we write sentences for our labels? to me, a complete sentence ends with a period and we don't push for that do we?

Reply:
I think in some places this guidance is followed and in others it isn't. It _is_ a little weird. This will be one of those things to revisit when we can chat with more stakeholders.